packages/helper-backend/src/langflow-mapping/dtos/create.dto.ts

```diff
@@ -1,14 +1,16 @@
-import { IsString, Validate } from 'class-validator';
-import { IsUniqueModel, IsUniqueModelRule } from '../pipes/unique';
+import { IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
 
 export class CreateLangFlowMapping {
+  @ApiProperty({ description: 'The model name' })
   @IsString()
-  @IsUniqueModel()
```

Comment:
We still want to make sure the model is unique when we make the mapping so this shouldn't be removed